contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -980,7 +969,14 @@ sp_describe_undeclared_parameters_internal(PG_FUNCTION_ARGS)
 		params = PG_ARGISNULL(1) ? NULL : text_to_cstring(PG_GETARG_TEXT_PP(1));
 
 		/* First, pass the batch to the ANTLR parser */
-		result = antlr_parser_cpp(batch);
+		if (batch)
+			result = antlr_parser_cpp(batch);
+		else
+		{
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
```

Comment:
Probably `ERRCODE_UNDEFINED_FUNCTION` would be better.

Reply:
Updated